Doc/library/sqlite3.rst

```diff
@@ -302,10 +302,10 @@ Module functions
    :type isolation_level: str | None
 
    :param bool check_same_thread:
-       If ``True`` (default), only the creating thread may use the connection.
-       If ``False``, the connection may be shared across multiple threads;
-       if so, write operations should be serialized by the user to avoid data
-       corruption.
+       If ``True`` (default), an exception will be raised if the database connection
+       is used by a thread other than the one that created it.
+       This may be helpful in particular when SQLite is not in serialized mode
+       (see :attr:`threadsafety`).
```

Comment:
Hm, I find this slightly more difficult to read than the original. This is a reference, so it should be as accurate and to the point as possible (please see [^1]). IMO, _"If False"_ is a better way to start this sentence because it is unambiguous and succinct.

I'm afraid I don't have a useful suggestion to come up with today (I'll sleep on it).

[^1]: https://diataxis.fr/reference/

Reply:
Yeah, IMO this line is a significant regression on the previous revision for unclear benefit, removing explicit description of what `False` does as well as other relevant and appropriate information, adopting a tone not consistent with use in a reference, and overall much less clear and more confusing to me at least as a non-expert. What's more, I'm not even sure this directly address @erlend-aasland 's original comment that prompted this. I'm sorry if I confused you with my feedback, as this was not at all what I suggesting.

I would suggest just reverting to the previous wording (with the "please see" fix), and considering if any of the ≈dozen much more focused suggestions offered previously, or some variant of the same, resolve the concern around "seriali

Here's that, as a baseline to start with, with a few much more conservative fixes and tweaks to start off with:

```suggestion
       If ``False``, the connection may be accessed in multiple threads;
       write operations may need to be serialized by the user
       to avoid data corruption.
       See :attr:`threadsafety` for more information.
```